Readme.md

```diff
@@ -28,11 +28,36 @@ As possible users, we see energy service companies (ESCOs) who want to build rea
 
 ## How does FlexMeasures enable rapid development of energy flexibility apps?
 
-A closer look at FlexMeasures' three core value drivers:
+FlexMeasures is designed to help with three basic needs of developers in the energy flexibility domain:
 
-1. Real-time data intelligence and integration, with advice for the rest of the day. For example, forecasts and schedules are made available via API (designed with [the USEF framework](https://usef.energy) in mind).
-2. Energy sensor and environment data have multiple sources and their forecasts are uncertain. FlexMeasures uses the [timely-beliefs library](https://github.com/SeitaBV/timely-beliefs) to model this well.
-3. Developer support ― building customer-facing apps & services is where energy flexibility hits the road. FlexMeasures reduces developer workload with a well-documented API, data visualisation and multi-tenancy, and it supports plugins to customise and extend the platform to your needs.
+### I need help integrating available real-time data and continuously compute new data
```

Comment:
- too many adjectives before _data_: I suggest removing _available_
- compute -> computing, or possibly rewrite as:
I need help with integrating real-time data and with continuously computing new data